examples/basic.mdx

```diff
@@ -11,7 +11,7 @@ party Receiver;
 tx send_ada(quantity: Int) {
     input source {
         from: Sender,
-        min_amount: Ada(quantity),
+        min_amount: Ada(quantity) + fees,
```

Comment:
Updated examples to clarify that `fees` should also be part of the `min_amount` to ensure the sender has enough balance